### ISSUE TITLE
[Bug] Failing to Render Dashboard Storybook

### ISSUE THREAD

**Issue by tylerferrara** [DimensionDev/Maskbook#5173]
## Bug Info
After building the dashboard storybook and hosting the static files, the UI is stuck in an endless loading screen (shown below).
![Storybook](https://user-images.githubusercontent.com/20875599/145760362-3976c6fd-0f90-4b6a-bff5-e025dc7df53f.png)

### How to reproduce?

1. Install the http-server tool (to host static files)
```zsh
$ pnpm install http-server -g
```
2. Build the dashboard storybook (within `Maskbook/packages/dashboard`)
```zsh
$ pnpm build-storybook
```
3. Host the static files
```zsh
$ http-server Maskbook/packages/dashboard/storybook-static
```
4. Navigate to `http://127.0.0.1:8080/` in chrome

### What happened?
It appears a request to `/stories.json` is causing this failure.

Chrome Dev Tools show:
<img width="740" alt="Screen Shot 2021-12-13 at 1 09 20 AM" src="https://user-images.githubusercontent.com/20875599/145761120-756bd70c-7525-4435-943d-8a18639911c8.png">

Output from `http-server`:
```zsh
Starting up http-server, serving storybook-static

http-server version: 14.0.0

http-server settings: 
CORS: disabled
Cache: 3600 seconds
Connection Timeout: 120 seconds
Directory Listings: visible
AutoIndex: visible
Serve GZIP Files: false
Serve Brotli Files: false
Default File Extension: none

Available on:
  http://127.0.0.1:8080
  http://172.16.42.42:8080
Hit CTRL-C to stop the server

[2021-12-13T06:13:11.661Z]  "GET /" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Safari/537.36"
(node:43178) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
(Use `node --trace-deprecation ...` to show where the warning was created)
[2021-12-13T06:13:11.991Z]  "GET /stories.json" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Safari/537.36"
[2021-12-13T06:13:11.992Z]  "GET /stories.json" Error (404): "Not found"
[2021-12-13T06:13:12.453Z]  "GET /483.7af154bc.iframe.bundle.js.map" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Safari/537.36"

```

### What should happen?
The storybook UI should load and present the documentation.

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [X] Build of version/commit hash: branch:`develop` commit:`8bf3d009ee14c3cd040835bc2a961ab017d287fd`

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [X] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [X] Latest Chrome (stable channel)
- [X] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

**Comment by tylerferrara** [comment]
Let me also add:
I am also experiencing an issue loading the storybook in the dev server.
```zsh
$ pnpm storybook &> out.txt
$ cat out.txt
info @storybook/react v6.3.12
info 
1:16:10 AM - Starting compilation in watch mode...

info => Loading presets
(node:43543) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/@storybook+ui@6.4.0-alpha.38_604986fcc13cdc21a00881f97a5a914a/node_modules/react/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
info => Loading custom manager config
info => Loading custom manager config
info => Using implicit CSS loaders
info => Using React fast refresh
info => Using default Webpack5 setup
<s> [webpack.Progress] 0% 

<s> [webpack.Progress] 1% setup initialize
<s> [webpack.Progress] 1% setup initialize HtmlWebpackPlugin
<s> [webpack.Progress] 1% setup initialize
(node:43543) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/@storybook+builder-webpack5@6.4.0-alpha.38_9c7ea15ad227adafb49b257fef99d61c/node_modules/react/package.json.
Update this package.json to use a subpath pattern like "./*".
<i> [webpack-dev-middleware] wait until bundle finished
<s> [webpack.Progress] 3% setup watch run
<s> [webpack.Progress] 3% setup watch run VirtualModulesPlugin
<s> [webpack.Progress] 3% setup watch run webpack-dev-middleware
<s> [webpack.Progress] 3% setup watch run
<s> [webpack.Progress] 4% setup normal module factory
<s> [webpack.Progress] 4% setup normal module factory CaseSensitivePathsPlugin
<s> [webpack.Progress] 4% setup normal module factory
<s> [webpack.Progress] 5% setup context module factory
<s> [webpack.Progress] 5% setup context module factory
<s> [webpack.Progress] 6% setup before compile
<s> [webpack.Progress] 6% setup before compile ProgressPlugin
<s> [webpack.Progress] 6% setup before compile ProgressPlugin
<s> [webpack.Progress] 6% setup before compile
<s> [webpack.Progress] 7% setup compile
<s> [webpack.Progress] 7% setup compile ExternalsPlugin
<s> [webpack.Progress] 7% setup compile
<s> [webpack.Progress] 8% setup compilation
<s> [webpack.Progress] 8% setup compilation ArrayPushCallbackChunkFormatPlugin
<s> [webpack.Progress] 8% setup compilation JsonpChunkLoadingPlugin
<s> [webpack.Progress] 8% setup compilation StartupChunkDependenciesPlugin
<s> [webpack.Progress] 8% setup compilation ImportScriptsChunkLoadingPlugin
<s> [webpack.Progress] 8% setup compilation FetchCompileWasmPlugin
<s> [webpack.Progress] 8% setup compilation FetchCompileAsyncWasmPlugin
<s> [webpack.Progress] 8% setup compilation WorkerPlugin
<s> [webpack.Progress] 8% setup compilation SplitChunksPlugin
<s> [webpack.Progress] 8% setup compilation RuntimeChunkPlugin
<s> [webpack.Progress] 8% setup compilation ResolverCachePlugin
<s> [webpack.Progress] 8% setup compilation HtmlWebpackPlugin
<s> [webpack.Progress] 8% setup compilation
<s> [webpack.Progress] 9% setup compilation
<s> [webpack.Progress] 9% setup compilation DefinePlugin
<s> [webpack.Progress] 9% setup compilation HotModuleReplacementPlugin
<s> [webpack.Progress] 9% setup compilation ProgressPlugin
<s> [webpack.Progress] 9% setup compilation ProvidePlugin
<s> [webpack.Progress] 9% setup compilation ReactRefreshPlugin
<s> [webpack.Progress] 9% setup compilation DocGenPlugin
<s> [webpack.Progress] 9% setup compilation ProvidePlugin
<s> [webpack.Progress] 9% setup compilation ChunkPrefetchPreloadPlugin
<s> [webpack.Progress] 9% setup compilation ModuleInfoHeaderPlugin
<s> [webpack.Progress] 9% setup compilation SourceMapDevToolPlugin
<s> [webpack.Progress] 9% setup compilation JavascriptModulesPlugin
<s> [webpack.Progress] 9% setup compilation JsonModulesPlugin
<s> [webpack.Progress] 9% setup compilation AssetModulesPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation EntryPlugin
<s> [webpack.Progress] 9% setup compilation RuntimePlugin
<s> [webpack.Progress] 9% setup compilation InferAsyncModulesPlugin
<s> [webpack.Progress] 9% setup compilation DataUriPlugin
<s> [webpack.Progress] 9% setup compilation FileUriPlugin
<s> [webpack.Progress] 9% setup compilation CompatibilityPlugin
<s> [webpack.Progress] 9% setup compilation HarmonyModulesPlugin
<s> [webpack.Progress] 9% setup compilation AMDPlugin
<s> [webpack.Progress] 9% setup compilation RequireJsStuffPlugin
<s> [webpack.Progress] 9% setup compilation CommonJsPlugin
<s> [webpack.Progress] 9% setup compilation LoaderPlugin
<s> [webpack.Progress] 9% setup compilation LoaderPlugin
<s> [webpack.Progress] 9% setup compilation NodeStuffPlugin
<s> [webpack.Progress] 9% setup compilation APIPlugin
<s> [webpack.Progress] 9% setup compilation ExportsInfoApiPlugin
<s> [webpack.Progress] 9% setup compilation WebpackIsIncludedPlugin
<s> [webpack.Progress] 9% setup compilation ConstPlugin
<s> [webpack.Progress] 9% setup compilation UseStrictPlugin
<s> [webpack.Progress] 9% setup compilation RequireIncludePlugin
<s> [webpack.Progress] 9% setup compilation RequireEnsurePlugin
<s> [webpack.Progress] 9% setup compilation RequireContextPlugin
<s> [webpack.Progress] 9% setup compilation ImportPlugin
<s> [webpack.Progress] 9% setup compilation SystemPlugin
<s> [webpack.Progress] 9% setup compilation ImportMetaPlugin
<s> [webpack.Progress] 9% setup compilation URLPlugin
<s> [webpack.Progress] 9% setup compilation DefaultStatsFactoryPlugin
<s> [webpack.Progress] 9% setup compilation DefaultStatsPresetPlugin
<s> [webpack.Progress] 9% setup compilation DefaultStatsPrinterPlugin
<s> [webpack.Progress] 9% setup compilation JavascriptMetaInfoPlugin
<s> [webpack.Progress] 9% setup compilation EnsureChunkConditionsPlugin
<s> [webpack.Progress] 9% setup compilation RemoveEmptyChunksPlugin
<s> [webpack.Progress] 9% setup compilation MergeDuplicateChunksPlugin
<s> [webpack.Progress] 9% setup compilation SideEffectsFlagPlugin
<s> [webpack.Progress] 9% setup compilation FlagDependencyExportsPlugin
<s> [webpack.Progress] 9% setup compilation FlagDependencyUsagePlugin
<s> [webpack.Progress] 9% setup compilation NamedModuleIdsPlugin
<s> [webpack.Progress] 9% setup compilation NamedChunkIdsPlugin
<s> [webpack.Progress] 9% setup compilation DefinePlugin
<s> [webpack.Progress] 9% setup compilation TemplatedPathPlugin
<s> [webpack.Progress] 9% setup compilation RecordIdsPlugin
<s> [webpack.Progress] 9% setup compilation WarnCaseSensitiveModulesPlugin
<s> [webpack.Progress] 9% setup compilation IgnoreWarningsPlugin
<s> [webpack.Progress] 9% setup compilation ProgressPlugin
<s> [webpack.Progress] 9% setup compilation
<s> [webpack.Progress] 10% building
<s> [webpack.Progress] 10% building 0/1 entries 0/0 dependencies 0/0 modules
(node:43543) [DEP_WEBPACK_MAIN_TEMPLATE_OUTPUT_OPTIONS] DeprecationWarning: MainTemplate.outputOptions is deprecated (use Compilation.outputOptions instead)
(node:43543) [DEP_WEBPACK_MAIN_TEMPLATE_REQUIRE] DeprecationWarning: MainTemplate.hooks.require is deprecated (use JavascriptModulesPlugin.getCompilationHooks().renderRequire instead)
(node:43543) [DEP_WEBPACK_COMPILATION_NORMAL_MODULE_LOADER_HOOK] DeprecationWarning: Compilation.hooks.normalModuleLoader was moved to NormalModule.getCompilationHooks(compilation).loader

info => Using cached manager
/Users/tyler/Projects/Maskbook/node_modules/.pnpm/enhanced-resolve@5.8.3/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:40
	if (error) throw error;
	           ^

Error: NormalModuleFactory.afterResolve (CaseSensitivePathsPlugin, ReactRefreshPlugin) is no longer a waterfall hook, but a bailing hook instead. Do not return the passed object, but modify it instead. Returning false will ignore the request and results in no module created.
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/webpack@5.64.4/node_modules/webpack/lib/NormalModuleFactory.js:293:14
    at _next0 (eval at create (/Users/tyler/Projects/Maskbook/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:17:1)
    at eval (eval at create (/Users/tyler/Projects/Maskbook/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:33:1)
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:176:9
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:125:7
    at CaseSensitivePathsPlugin.fileExistsWithCase (/Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:95:5)
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:118:10
    at /Users/tyler/Projects/Maskbook/node_modules/.pnpm/case-sensitive-paths-webpack-plugin@2.4.0/node_modules/case-sensitive-paths-webpack-plugin/index.js:72:5
    at runCallbacks (/Users/tyler/Projects/Maskbook/node_modules/.pnpm/enhanced-resolve@5.8.3/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:34:4)
 ELIFECYCLE  Command failed with exit code 1.

1:16:20 AM - Found 0 errors. Watching for file changes.
```

**Comment by yisiliu** [comment]
@Jack-Works do you still maintain the storybook?

**Comment by Jack-Works** [comment]
Hi! Thanks for your report. I can confirm the dashboard storybook is broken and I'm investigating why.

**Comment by Jack-Works** [comment]
fixed in bddc1eedf5e2502a5f64be117ffdebd39a393a86.

Notice that now the dashboard and theme still don't render right with the MUI theme.

You should use `pnpm run storybook -C ./packages/dashboard/` or `pnpm run build-storybook -C ./packages/dashboard/` to start the storybook.

Thanks for your interest!

**Comment by tylerferrara** [comment]
> fixed in [bddc1ee](https://github.com/DimensionDev/Maskbook/commit/bddc1eedf5e2502a5f64be117ffdebd39a393a86).
> 
> Notice that now the dashboard and theme still don't render right with the MUI theme.
> 
> You should use `pnpm run storybook -C ./packages/dashboard/` or `pnpm run build-storybook -C ./packages/dashboard/` to start the storybook.
> 
> Thanks for your interest!

I will modify the README so others don’t bump into this issue.
